libvmi/os/linux/symbols.c

```diff
@@ -210,6 +210,17 @@ char* linux_system_map_address_to_symbol(
     return NULL;
 }
 
+char* linux_system_map_address_to_symbol_kaslr(
```

Comment:
Do we need a separate function for this? I think the kaslr offset subtraction could just be moved into linux_system_map_address_to_symbol directly.